src/client/app/components/Content.ts

```diff
@@ -4,15 +4,16 @@ import { useRoute } from '../router.js'
 export const Content = defineComponent({
   name: 'VitePressContent',
   props: {
-    onContentUpdated: Function
+    onContentUpdated: Function,
+    as: { type: [Object, String], default: 'div' }
```

Comment:
I usually use `tag` for this type prop, but is there any standard for this? I've looked into Vuetify but couldn't find props that changes the element type 🤔 

It would be cool if we can follow some other ui libs for this kind of stuff to just align with them.

If we can't find it, I think we can go with `as`.

@brc-dd Happen to know any?

Reply:
HeadlessUI uses `as` just like this, I thought it was standard at first.

Reply:
Yeah, `as` is fine IG. It's more common in React land.